shared/facebook/facebook.properties.po

```diff
@@ -68,7 +67,7 @@ msgstr ""
 
 #: linkProposal[zero]
 msgid "No matching friends. Showing all"
-msgstr ""
+msgstr "akula bangane abahambelana. sitshengisa bonke"
```

Comment:
Fix case. I think this should read "Akula bangane abahambelana lalokhu okubhalileyo. Sesizatshengisa bonke"